### PR TITLE
Edited solas documents

### PR DESCRIPTION
**Pull request by NancyHarvey**
Wordsmithed and updated some screenshots. I still need to make some inline comments though (apparently can't until I create the PR). 

Forewarning: I used HTML code on some images (see Slack and Jenkins for example) ... and used URL of my fork. May have to adjust after merge.

Suggest wherever possible to show steps by annotating screenshots (see some examples in Jenkins), instead of bulleted lists followed by a screenshot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
-SOLAS is also an international maritime treaty to ensure ships comply with
-minimum safety standards in construction, equipment and operation.
+At Samsung CNCT, solas refers to our minimum standards for operating in our sea of continuous cloud-native integration and development. 
 
-# Quickstart
+We use Jenkins to implement our CI/CD pipelines, with one Jenkins job for
+each GitHub repository. Each job builds, tests and then deploys an artifact
+to Quay. Here you'll find our best practices for creating and managing GitHub repositories, templates for issues and PRs, commit hooks, licensing guidelines and more.
+
+## Quickstart
```
Comment (NancyHarvey) [inline README.md]: this could use an intro ... sorry, I honestly wasn't clear on what to write. Maybe "To get started contributing to a new project, follow the steps below."

Maybe we should have links to all the docs in this repo instead or in addition though (Slack, GitHub, Jenkins ...)?

File: docs/quay.md [hunk 2]
```diff
@@ -1,85 +1,85 @@
...
-* Select `Application Repository`, for `solas-chart` derived repositories
+* Select `Container Repository` for `solas-container` repositories
 
-* Choose `samsung_cnct` for the namespace.
+* Select `Application Repository` for `solas-chart` repositories
 
-* Enter the name of the artifact. For example, `zabra-container` or `zabra`.
+* Choose `samsung_cnct` for the namespace
+
+* Enter the name of the artifact (for example, `zabra-container` or `zabra`)
```
Comment (NancyHarvey) [inline docs/quay.md]: The screenshots on this page could use some work. They don't exactly align with the accompanying text and are a bit hard to follow. Consider annotating screenshots ... can discuss, if easier.

File: README.md [hunk 3]
```diff
@@ -1,25 +1,23 @@
 # solas
-Solas is scaffolding for new repositories hosted by Samsung CNCT. It
-implements our best practices, such as issue and PR templates, commit hooks,
-licensing guidelines, and so on.
```
Comment (DStorck) [inline README.md]: we probably want some sort of intro like this.

File: docs/jenkins.md [hunk 4]
```diff
@@ -53,37 +50,43 @@ sources:
 - https://github.com/samsung-cnct/chart-zabra
 ```
 
-Also add any relevant keywords. For inspiration you should look at other
-related charts. For example, if you are creating a logging chart, you might
-look at https://github.com/samsung-cnct/chart-fluent-bit.
+Also add any relevant keywords and look at other
+related charts for inspiration. For example, if you're creating a logging chart, you might
+look at the [Fluent Bit CHart](https://github.com/samsung-cnct/chart-fluent-bit).
```
Comment (DStorck) [inline docs/jenkins.md]: Chart has H capitalized

File: docs/jenkins.md [hunk 5]
```diff
@@ -53,37 +50,43 @@ sources:
...
-  * Using anonymous access to github may result in throttling
-* _Select your repository, e.g. "container-zabra", "chart-zabra", etc._
-* _Under the GitHub branch source, add the additional behaviors "Advanced
-checkout behaviors" and "Advanced clone behaviors", but do not change their
-settings._
+Notes:
+* Once you choose a credential, you may need to go back and select it.
+* Under the GitHub Branch Source, add the additional behaviors "Advanced
+checkout behaviors" and "Advanced clone behaviors," but don't change their
+settings.
```
Comment (DStorck) [inline docs/jenkins.md]: The pic doesn't actually show these two advanced checkout behaviors, and actually mentions one that we don't want, "add name filter (with regex)" - unless I'm mistaken and someone mentioned that we are now including this

File: docs/jenkins.md (hunk 4 above)
Comment (NancyHarvey) [reply docs/jenkins.md]: thanks!

File: docs/jenkins.md (hunk 5 above)
Comment (NancyHarvey) [reply docs/jenkins.md]: Yeah - agree. I forgot to add an inline comment after I made the commit. Had previously asked @guineveresaenger offline for input. Can someone Slack me a screenshot that will match the text and I will update?

File: docs/quay.md (hunk 2 above)
Comment (NancyHarvey) [reply docs/quay.md]: @coffeepac - who's best to help with this?

File: README.md (hunk 3 above)
Comment (NancyHarvey) [reply README.md]: Thanks - adjusted

File: docs/quay.md (hunk 2 above)
Comment (NancyHarvey) [reply docs/quay.md]: @davidewatson or @DStorck - can one of you help with this?

**Comment by NancyHarvey** [comment]
@davidewatson - just waiting for updated screenshots and copy for Robots section of Quay as we Slacked about on Monday. Want to tackle today?

**Comment by davidewatson** [comment]
We are going to be using this documentation to create new repositories this week. Another will be opened based on additional feedback.
